src/FINDOLOGIC/Export/Data/Usergroup.php

```diff
@@ -33,11 +33,11 @@ public function getDomSubtree(\DOMDocument $document)
     }
 
     /**
-     * No-op, because usergroups are not supported in the CSV export format.
+     * @inheritdoc
      */
-    public function getCsvFragment()
+    public function getCsvFragment(array $availableProperties = [])
     {
-        return '';
+        return $this->value;
     }
 
     public function __toString()
```

Comment:
Is this function actually in use? Otherwise drop!

Reply:
It's useful for debugging, so I'll keep it.